.github/CODEOWNERS

```diff
@@ -1,29 +1,41 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
+# This file marks who is the "owner" of several files and directory on this GitHub
+# repository. This is mostly used to assign reviews on new PRs.
+#
+# - Directories and files which are assigned to a team should be interepreted as
+#   "collectively owned". GitHub will automatically assign members of the team
+#   to review the PR, in a round-robin fashion.
+# - Directories and files which are owned by one specific user X are to be
+#   interpreted as "primarily maintained by X". Any significant changes to
+#   that given code should be reviewed by X; as if it were another repository
+#   where X is the primary developer. However, ownership here can be ignored if
```

Comment:
Should we suggest to the individual code owner to unassign themselves if the change is small enough?